WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewModel.swift

```diff
@@ -163,6 +182,44 @@ class InPersonPaymentsMenuViewModel: ObservableObject {
 
     func learnMoreTapped(from viewController: UIViewController) {
         WebviewHelper.launch(learnMoreURL, with: viewController)
+        analytics.track(.paymentsHubCashOnDeliveryToggleLearnMoreTapped)
+    }
+}
+
+// MARK: - Analytics
+extension InPersonPaymentsMenuViewModel {
```

Comment:
I see all methods within the extension are private so there would no change, but, should we make the extension private instead?

Reply:
Yeah I can do this in a follow up as well. I think at one point I had a single public method in here that I deleted or made private.